packages/jest-haste-map/src/index.ts

```diff
@@ -466,7 +457,7 @@ export default class HasteMap extends EventEmitter {
     const setModule = (id: string, module: ModuleMetaData) => {
       let moduleMap = map.get(id);
       if (!moduleMap) {
-        moduleMap = Object.create(null) as {};
+        moduleMap = Object.create(null) as ModuleMapItem;
```

Comment:
do we need to cast? isn't `moduleMap` already the correct type?

Reply:
Does not work without casting. It needs some type, because `Object.create()` returns `any`.